js/ui/paygUnlockDialog.js

```diff
@@ -20,22 +38,22 @@ const Monitor = imports.ui.monitor;
 const ShellEntry = imports.ui.shellEntry;
 const Tweener = imports.ui.tweener;
 
-// The timeout before going back automatically to the lock screen (in seconds)
-const IDLE_TIMEOUT = 2 * 60;
+const ONE_SECOND = 1000
```

Comment:
Nitpick: I’d call this something like `MSEC_PER_SEC`, partly to fit in with GLib’s `USEC_PER_SEC`, and partly to make it clear that this is in milliseconds.

Reply:
Sounds good. I used ONE_SECOND for consistency with https://github.com/endlessm/gnome-shell/blob/519a389dae2f1b8c7fc28b944b4f6966f8dcb359/js/ui/windowManager.js#L46, but I like more your proposed alternative.